package.json

```diff
@@ -2,8 +2,8 @@
   "name": "numbers_api",
   "version": "1.0.0",
   "scripts": {
-    "test": "jest",
-    "start": "NODE_ENV=development nodemon app.js",
+    "test": "jest --forceExit",
+    "start": "NODE_ENV=development node app.js",
```

Comment:
do we need these changes?

Reply:
Removed the `--forceExit` flag. I changed the `start` script to use `node` instead of `forever`.

Reply:
Closing ticket #35 and implementing the changes in this ticket. That fixes the `--forceExit` issue.